dev/bots/test.dart

```diff
@@ -272,12 +272,10 @@ Future<void> _runBuildTests() async {
       continue;
     }
     final String examplePath = fileEntity.path;
-    final String basename = path.basename(examplePath);
-    final bool expectIpaBuildFailure = basename == 'platform_channel_swift'; // Remove when https://github.com/flutter/flutter/issues/35773 is fixed.
 
     await _flutterBuildAot(examplePath);
     await _flutterBuildApk(examplePath);
-    await _flutterBuildIpa(examplePath, expectIpaBuildFailure: expectIpaBuildFailure);
+    await _flutterBuildIpa(examplePath);
```

Comment:
Also could you remove the expectIpaBuildFailure parameter in _flutterBuildIpa since now it's unused?

Reply:
Done.